Metalama.Framework.DesignTime/ProjectHandler.cs

```diff
@@ -26,6 +26,20 @@ public abstract class ProjectHandler : IDisposable
 
     protected ILogger Logger { get; }
 
+    /// <summary>
+    /// Gets an override of touch id for the current project, for the cases when the IDE is not watching the touch file for changes.
+    /// </summary>
+    /// <remarks>
+    /// This exists for two reasons:
+    /// 1. Roslyn in VS has a bug (https://github.com/dotnet/roslyn/issues/74716) where the touch file is not correctly watched for changes.
+    /// 2. Rider does not currently watch the touch file for changes (https://youtrack.jetbrains.com/issue/RIDER-75959).
+    /// 
+    /// Note that this approach can't trigger running the generator, which means that changes will become visible only once the user types something in some C# file in the project.
+    /// 
+    /// Also, it only works in the current process.
+    /// </remarks>
+    internal string? TouchIdOverride { get; private protected set; }
+
```

Comment:
Rename to LastTouchId.

The comment above must be moved to where the property is used.